### PR TITLE
shell: Use manifest to specify machine limits

### PR DESCRIPTION
**Pull request by petervo**
Fixes a RHEL regression

### CODE REVIEW

**Comment by stefwalter** [comment]
```
./rhel-7.install: line 8: /usr/share/cockpit/shell/override.json: No such file or directory
```

**Comment by petervo** [comment]
Depends on #3485

**Comment by petervo** [comment]
Thanks to @mvollmer this is passing now.

File: pkg/shell/machine-dialogs.js [hunk 1]
```diff
@@ -424,18 +425,10 @@ define([
...
-                try {
-                    var doc = window.parent.document;
-                    limit_el = $('.machine-limit', doc);
-                } catch (ex) {}
-            }
-
-            var limit = parseInt(limit_el.text(), 10);
-            if (isNaN(limit)) {
+            var manifest = local_manifests["shell"] || {};
+            var limit = parseInt(manifest["machine-limit"]);
```
Comment (stefwalter) [inline pkg/shell/machine-dialogs.js]: This needs a ```10``` as the second argument.

Comment (stefwalter) [inline pkg/shell/machine-dialogs.js]: Will make this change when merging.

**Pull request closed without merging**